README.md

```diff
@@ -134,6 +134,13 @@ of the most common ones with non default values. For more info take a look at th
   Token used to verify the device identity with Google.
   See more in https://developers.google.com/identity/protocols/OAuth2
 
+#### File Storage System (Used on Kernel, ODK and UI Modules)
+- `DJANGO_REMOTE_STORAGE` (optional): Used to specify a custom [Default file storage system](https://docs.djangoproject.com/en/2.0/ref/settings/#std:setting-DEFAULT_FILE_STORAGE). If not provided, defaults to the [Django default storage](https://docs.djangoproject.com/en/2.0/ref/files/storage/#django.core.files.storage.FileSystemStorage) Available options: S3, . More information [here](https://django-storages.readthedocs.io/en/latest/index.html)
+- `REMOTE_STATIC_FILES` (optional): Set to "true" to collect static files to the selected `DJANGO_REMOTE_STORAGE`
+- `BUCKET_NAME` (optional): Name of the bucket to use for `DJANGO_REMOTE_STORAGE`. Must be unique on the storage provider. If not provided, it defaults to the `HOSTNAME` value. Bucket is automatically created if does not exist.
+- `AWS_ACCESS_KEY_ID`: AWS Access Key to your S3 account. Used when `DJANGO_REMOTE_STORAGE=S3`.
```

Comment:
we should never need to use AWS keys as we use AWS IAM roles for this

Reply:
see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html

Reply:
@MrPink Will that be the case if aether is hosted outside AWS?

Reply:
![image](https://user-images.githubusercontent.com/63928/43406096-cbaeb9e6-941b-11e8-9491-45f707ccc3ed.png)

We have options

Reply:
it should use the supported `filesystem-storage-class` for that particular Cloud provider

Reply:
I understand what you mean. What is implemented allows a developer to choose to host on one platform and store data on another. Even if the host isn't listed above